google-cloud-compute/src/main/resources/META-INF/native-image/resource-config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This configuration is supposed to be brought in from gax but it isn't getting picked up by the tests. Same for `GoogleJsonClientFeature`:

```
 - com.google.api.gax.grpc.nativeimage.GrpcNettyFeature
  - com.google.api.gax.grpc.nativeimage.ProtobufMessageFeature
  - com.google.cloud.nativeimage.features.clients.CloudFunctionsFeature
  - com.google.cloud.nativeimage.features.clients.CloudSqlFeature
  - com.google.cloud.nativeimage.features.clients.SpannerFeature
  - com.oracle.svm.thirdparty.gson.GsonFeature
  - org.graalvm.junit.platform.JUnitPlatformFeature
```

Reply:
This comment is outdated. The CI includes GAX configurations.